src/univariate/continuous/noncentralchisq.jl

```diff
@@ -1,3 +1,24 @@
+"""
+    NoncentralChisq(ν, λ)
+
+The *noncentral chi-squared distribution* with `ν` degrees of freedom and noncentrality parameter `λ` has the probability density function
+
+$f(x; ν, λ) = \frac{1}{2}e^{-(x + λ)/2}\frac{x}{λ}^{ν/4-1/2}I_{ν/2-1}(\sqrt{λx}) \quad x>0.$
+
+It is the distribution of the sum of squares of `ν` independent [`Normal`](:func:`Normal`) variates with individual means $μ_i$ and
+
+$λ = \sum_{i=1}^ν{μ_i^2}$
```

Comment:
Thanks for this tip. TeX isn't my forte and I was wondering which was the better way to use unicode in it.